### PR TITLE
tests : Add support for Z

### LINKED ISSUE #669: Add support for Z

### PR DESCRIPTION
**Pull request by ydjainopensource**
Fixes #669

Signed-off-by: Yash Jain <ydjainopensource@gmail.com>

### CODE REVIEW

File: .ci/kata-arch.sh [hunk 1]
```diff
@@ -38,7 +38,8 @@ arch_to_golang()
 		aarch64) echo "arm64";;
 		ppc64le) echo "$arch";;
 		x86_64) echo "amd64";;
-		*) die "unsupported architecture: $arch";;
+        s390x) echo "s390x";;
```
Comment (jodh-intel) [inline .ci/kata-arch.sh]: Please could you sort out the indent (I'm guessing it might be a "tabs vs. spaces" issue here?)

**Comment by jodh-intel** [comment]
Once the indent issue is resolved,

lgtm

CI kicked (network timeout).

**Comment by grahamwhaley** [comment]
l g t m once the indent fixes are done.
all the CIs seem very upset, but when I went to check 16.04, it was a soak test fail?
```
Checking 0 containers have all relevant components
Run kata-runtime: nginx: 
Checking 1 containers have all relevant components
Wrong number of containers running (0 != 1) - stopping
Showing system state:
 --Docker ps--
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
 --kata-runtime list--
ID          PID         STATUS      BUNDLE      CREATED     OWNER
 --pgrep kata-proxy--
 --pgrep kata-shim--
 --pgrep kata-runtime--
 --pgrep qemu--
 --pgrep ksm-throttler--
Makefile:62: recipe for target 'docker-stability' failed
make: *** [docker-stability] Error 1
```

I think we wait for the indent repush and see how they look then - this feels somewhat unrelated to the PR :-)

File: .ci/kata-arch.sh (hunk 1 above)
Comment (ydjainopensource) [reply .ci/kata-arch.sh]: I didn't set my vimrc properly just added `set autoindent noexpandtab tabstop=4 shiftwidth=4` Hopefully this should do the job

**Comment by jodh-intel** [comment]
Thanks @ydjainopensource!

**Comment by devimc** [comment]
CI is not happy, @ydjainopensource  can you re-push your changes to restart the CI?

**Comment by ydjainopensource** [comment]
@devimc reckicked the CI's

**Comment by ydjainopensource** [comment]
@devimc The build's failing due to something else.
Can this be merged today? I need this to get the CI running.

**Comment by ydjainopensource** [comment]
@devimc Thanks a lot

**Pull request merged**